src/test/java/org/ladocuploader/app/submission/actions/ValidateContactMethodTest.java

```diff
@@ -0,0 +1,126 @@
+package org.ladocuploader.app.submission.actions;
+
+import formflow.library.data.FormSubmission;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest()
+@ActiveProfiles("test")
+class ValidateContactMethodTest {
+
+    private final String VALID_PHONE="(123) 459-0392";
+    private final String VALID_EMAIL="test@mail.com";
+    private final String EMAIL_REMINDER="By Email";
+    private final String PHONE_REMINDER="By Text";
+    private final String BLANK_VALUE="";
+    private final String PHONE_NUMBER_INPUT_NAME = "phoneNumber";
+    private final String EMAIL_ADDRESS_INPUT_NAME = "emailAddress";
+
+    @Autowired
+    private ValidateContactMethod validator;
+
+    @Test
+    public void testPhoneInvalidRaisesAnError() {
+        FormSubmission form = new FormSubmission(Map.of(
+                "phoneNumber", "1234561234",
+                "emailAddress", BLANK_VALUE,
+                "remindersMethod[]", new ArrayList<>(List.of(PHONE_REMINDER))
```

Comment:
I think you can just say "List.of(PHONE_REMINDER)" rather than passing that into the `new ArrayList<>()`

```
"remindersMethod[]",  List.of(PHONE_REMINDER)
```

Reply:
There are a few other spots where you do this, but I won't make a comment at each one.

Reply:
We were running into an issue when just passing List.of because it's immutable

Reply:
+1 just confirmed that we see the immutable issue when trying to pass just the List.